src/tests/ftest/dfuse/read.py

```diff
@@ -5,11 +5,12 @@
 
 import time
 
-from dfuse_test_base import DfuseTestBase
+from apricot import TestWithServers
+from dfuse_utils import get_dfuse, start_dfuse
 from run_utils import run_remote
 
 
-class DFusePreReadTest(DfuseTestBase):
+class DFuseReadTest(TestWithServers):
```

Comment:
Need to also update the tag, and can see the lint check failing:
https://github.com/daos-stack/daos/actions/runs/8928997496/job/24525781319

Reply:
Actually I need to update the class name.  This resulted from a bad merge of https://github.com/daos-stack/daos/commit/31b0de58f80bc025f4e39bb28859f15c38484e72 where the classname was changed.